nixos/modules/services/databases/postgresql.nix

```diff
@@ -80,6 +82,23 @@ in
         '';
       };
 
+      groupAccess = mkOption {
+        type = types.bool;
+        default = false;
+        example = true;
+        description = ''
+          Allow read access for group (0750 mask for data directory).
+          Supported only for PostgreSQL 11+. PostgreSQL 10 and lower doesn't
+          support starting server with 0750 mask, but a workaround like
```

Comment:
IMHO, we shouldn't encourage such hacks in the official documentation.
This workaround for PostgreSQL <= 10 should probably go on the wiki instead.